shared/components/home/card.tsx

```diff
@@ -1,4 +1,5 @@
 import { motion } from 'framer-motion'
+import Link from 'next/dist/client/link'
```

Comment:
@AdityaGupta0001 why are you importing the changes from next/dist/client/link? did you try next/link? https://nextjs.org/docs/pages/api-reference/components/link

Reply:
So sorry bhaiya, actually I just directly used Link tag and it automatically imported it as next/dist/client/link. Ill fix it now.

Reply:
Done bhaiya